packages/serverpod/lib/src/database/bulk_data.dart

```diff
@@ -128,9 +127,9 @@ class DatabaseBulkData {
       var duration = DateTime.now().difference(startTime);
 
       return BulkQueryResult(
-        headers: result.columnDescriptions
+        headers: result.schema.columns
             .map((e) => BulkQueryColumnDescription(
-                  name: e.columnName,
+                  name: e.columnName ?? '',
```

Comment:
Question: when can this be null? Seems strange to set the name to an empty string, if this is not suppose to happen I think it would be better to throw.

Reply:
I think this can be null when we perform queries like `SELECT 1;`. Since this is used in isights to display the name of the column I think it is fine to ensure that when we don't have a column name we provide an empty string instead.